src/output.cpp

```diff
@@ -218,7 +218,9 @@ std::string Output::get_helper_error_msg(int func_id, int retcode) const
 {
   std::string msg;
   if (func_id == libbpf::BPF_FUNC_map_update_elem && retcode == -E2BIG) {
-    msg = "Map full; can't update element. Try increasing max_map_keys config";
+    msg = "Map full; can't update element. Try increasing max_map_keys config "
```

Comment:
*cough* Hint *cough*

Reply:
Let me refactor this a little in a follow-up PR. I'll have to pass a tuple/pair/struct up to the caller.